### ISSUE TITLE
Dalaran-WoW filter: URL detected - not allowed

### ISSUE THREAD

**Issue by boujii** [dalaranwow/dalaran-wow#5533]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

**Expected behaviour**: Tell us what should happen instead

**Steps to reproduce the problem**:

1. When trying to link the Enchanting profession it results in Dalaran-WoW filter: URL detected - not allowed
2. I am able to link individual links from the Enchanting Profession though
3. I have deleted all addons, cache, wtf folder.
4. Downloaded fresh copy of the World of Warcraft Client 3.3.5a - Ready to Play and same results.
![screenshot_4](https://user-images.githubusercontent.com/1514895/42009057-9ce5a982-7a56-11e8-838f-e150e5e24fbb.png)
![screenshot_5](https://user-images.githubusercontent.com/1514895/42009058-9cf0e220-7a56-11e8-84bc-fd138385ed01.png)

Character: Boujii
Realm: Algalon

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by boujii** [comment]
This sucks because I have been trying to link my enchants to level up for a while now.

**Comment by boujii** [comment]
Yup still sucks. Makes no sense to even have a profession if you can't advertise it to others.

**Comment by Oreodog** [comment]
This isn't a bug, just Loth's shitty chat filter. It doesn't it for random, unlucky individuals, regardless of profession, in addition to random items and a variety of non-gold selling phrases.

**Comment by boujii** [comment]
It definitely sucks! I spent a lot of time and gold leveling up my enchanting. I am trying to get some rebound from it and can't. It's not hit and miss either. It's constant.

**Comment by TeridanDW** [comment]
Workaround: Link something before the enchanting like a spell from your spellbook with or without a space between them. That should help for the timebeen.

**Comment by boujii** [comment]
I had high hopes that workaround would work. I tried with and without spaces, before and after the link, all sorts of shit. No matter what, when I link my enchanting, it errors with the URL. This is bullshit!

**Comment by lkrocek** [comment]
Yes lot of items are not linkable to delegate loot by Master loot system have often problem  `/rw [item link] Main spec roll`
If I remember correctly this item was problematic to link in chat: https://wotlk.evowow.com/?item=40064
http://wowwiki.wikia.com/wiki/ItemLink

**Comment by boujii** [comment]
I wouldn't complain so much if it was items. But there should never be an issue when trying to link a profession.

**Comment by Golganeth** [comment]
You can link professions in-game. Make sure that it's not an addon or some text interfering.

**Comment by lkrocek** [comment]
I can confirm it now item of ID 40064 is not link-able, and I tried it also on kirin'tor , where this link is working properly!

**Comment by boujii** [comment]
Golganeth
You can link professions in-game. Make sure that it's not an addon or some text interfering.

In my original post I stated that I even downloaded the WoW 3.3.5a from the forum with no addons at all. And it does not do it with my alts on a different account. Just my main with enchanting. My other profession is mining which I don't think is linkable. But I will try cooking tonight and see if that does it too.

**Comment by LoveDalaranWow** [comment]
A lot of items are still censored by this script

**Comment by boujii** [comment]
I could live without being able to post items. But not being able to link professions it total crap! I spent tons of time, energy, and money into maxing out my enchanting and can't even link it to anyone. The whole point of that was to be able to get some return back for my investment.

**Comment by boujii** [comment]
Well after nearly 3 years this still sucks lol. Still unable to link my enchanting or jewelcrafting from my Boujii toon. I can link every other profession from all other toons using the exact same client. It's like the server has a ban on this toon from posting my professions. I can link individual jc and enchants from the profession, but not the main. So it spend tons of time getting 95% of all the recipes and formulas and can't even link it. Seriously messed up!
Again I downloaded a clean client with no addons and same thing. And only with this account.
